### PR TITLE
ocr: disable_tesseract if package is missing

### PR DESCRIPTION
**Pull request by sgliner-ledger**

### CODE REVIEW

**Review by lpascal-ledger — APPROVED** [review]
The code looks functionally OK, but the CI squeaks :p

File: speculos/main.py [hunk 1]
```diff
@@ -433,8 +439,12 @@ def main(prog=None):
     if api_enabled:
         apirun = ApiRunner(args.api_port)
 
+    if disable_tesseract: 
+        args.disable_tesseract = True
+
     display_args = display.DisplayArgs(args.color, args.model, args.ontop, rendering,
-                                       args.keymap, zoom, x, y, args.force_full_ocr)
+                                       args.keymap, zoom, x, y, args.force_full_ocr,
+                                       args.disable_tesseract)
```
Comment (xchapron-ledger) [inline speculos/main.py:447]: Why not directly using `disable_tesseract` here? You don't need to set it in args I think?

Comment (xchapron-ledger) [reply speculos/main.py:447]: my bad, I didn't see that you added an option for that...

File: speculos/main.py [hunk 2]
```diff
@@ -213,10 +213,15 @@ def setup_logging(args):
...
-        error_message = "tesseract-ocr is not found and is required to run Speculos.\n"
-        error_message += "Please run `sudo apt install tesseract-ocr`"
-        raise RuntimeError(error_message)
+        disable_tesseract = True
+        warning_message = "\n\n\n!****************************************************************!\n"
+        warning_message += "tesseract-ocr is not found and is required to run Speculos with ocr.\n"
+        warning_message += "Please run `sudo apt install tesseract-ocr`\n"
+        warning_message += "Speculos will continue without tesseract-ocr enabled\n"
+        warning_message += "!****************************************************************!\n\n\n"
+        print(warning_message)
```
Comment (xchapron-ledger) [inline speculos/main.py:224]:
```suggestion
        logger.warn(warning_message)
```

**Review by xchapron-ledger — APPROVED** [review]

File: speculos/main.py (hunk 2 above)
Comment (sgliner-ledger) [reply speculos/main.py]: Done :)

**Review by xchapron-ledger — APPROVED** [review]

**Pull request merged**